File: Documentation/4.0/Samples/java/src/test/java/net/ravendb/ClientApi/Configuration/Cluster.java
package net.ravendb.ClientApi.Configuration;

import net.ravendb.client.documents.conventions.DocumentConventions;
import net.ravendb.client.http.ReadBalanceBehavior;

public class Cluster {
    public Cluster() {

        DocumentConventions conventions = new DocumentConventions();

        //region ReadBalanceBehavior
        conventions.setReadBalanceBehavior(ReadBalanceBehavior.FASTEST_NODE);
        //endregion
    }
}

File: Documentation/5.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Configuration/Cluster.cs
using Raven.Client.Documents;
using Raven.Client.Documents.Conventions;
using Raven.Client.Http;

namespace Raven.Documentation.Samples.ClientApi.Configuration
{

    public class Cluster
    {
        public Cluster()
        {
            var store1 = new DocumentStore()
            {
                #region ReadBalanceBehavior
                Conventions = new DocumentConventions
                {
                    ReadBalanceBehavior = ReadBalanceBehavior.FastestNode
                }
                #endregion
            };


            var store2 = new DocumentStore()
            {
                #region LoadBalanceBehavior
                Conventions = new DocumentConventions
                {
                    LoadBalanceBehavior = LoadBalanceBehavior.UseSessionContext
                }
                #endregion
            };
        }

    }
}
